src/help/index.ts

```diff
@@ -197,55 +184,60 @@ export class Help extends HelpBase {
       command.id = command.id.replace(/:/g, this.config.topicSeparator)
       command.aliases = command.aliases && command.aliases.map(a => a.replace(/:/g, this.config.topicSeparator))
     }
-    const help = new CommandHelp(command, this.config, this.opts)
+    const help = new this.CommandHelpClass(command, this.config, this.opts)
     return help.generate()
   }
 
   protected formatCommands(commands: Interfaces.Command[]): string {
     if (commands.length === 0) return ''
 
-    const body = renderList(commands.map(c => {
+    const body = this.renderList(commands.map(c => {
       if (this.config.topicSeparator !== ':') c.id = c.id.replace(/:/g, this.config.topicSeparator)
       return [
         c.id,
-        c.description && this.render(c.description.split('\n')[0]),
+        this.summary(c),
       ]
     }), {
       spacer: '\n',
       stripAnsi: this.opts.stripAnsi,
-      maxWidth: this.opts.maxWidth - 2,
+      indentation: 2,
     })
 
-    return [
-      bold('COMMANDS'),
-      indent(body, 2),
-    ].join('\n')
+    return this.section('COMMANDS', body)
+  }
+
+  protected summary(c: Interfaces.Command): string {
+    if (c.summary) return this.render(c.summary.split('\n')[0])
+
+    return this.render((c.description || '').split('\n')[0])
```

Comment:
I believe this should be `c.summary` but for some reason the `.summary` isn't being added to the command. See comment on [line 118](https://github.com/oclif/core/pull/184/files#diff-ee5d33646fe2e22a534294dd09bb1a83b79742e98b3dc681175fe5754427b5f9R118)
```suggestion
    return this.render((c.summary|| '').split('\n')[0])
```

Reply:
I don't know if this is related but `.\bin\dev commandreference generate` seems to not be able to find `.name` [in here](https://github.com/forcedotcom/plugin-command-reference/blob/d7b6ef630cb383bbd963cb095f73c0a0ad1c1590/src/docs.ts#L145) when this branch is `yarn link`ed.

So it's like it's not getting the `.plugin` property [here](https://github.com/oclif/core/pull/184/files#diff-1212ed35aabbfc158720210527b07c5e3ccaf889121fa334acc54b160afe561cR70).